presets/4.4/tune/uav_tech/UAV_tech_6-7in.txt

```diff
@@ -1,4 +1,4 @@
-#$ TITLE: UAV Tech - 6-7in (~700g AUW)
+#$ TITLE: UAV Tech - 6"/7" (750-1000g AUW)
 #$ FIRMWARE_VERSION: 4.4
 #$ CATEGORY: TUNE
 #$ STATUS: COMMUNITY
```

Comment:
this has lots of changes, better change to community, at least till a few people get to test it as it is now

Reply:
just a file rename and change of title